org.jitsi.jitsi-meet.yaml

```diff
@@ -0,0 +1,60 @@
+---
+app-id: org.jitsi.jitsi-meet
+base: org.electronjs.Electron2.BaseApp
+base-version: '19.08'
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: jitsi-meet-run
+separate-locales: false
+tags: []
+finish-args:
+  - --device=all
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=x11
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: unappimage
+    buildsystem: simple
+    build-commands:
+      - make -C squashfs-tools -j ${FLATPAK_BUILDER_N_JOBS} install INSTALL_DIR=/app/bin
+    sources:
+      - type: git
+        url: https://github.com/refi64/unappimage
+        commit: d7f86f2a0d7ec3a69211125207d5f127386b849a
+
+  - name: jitsi-meet
```

Comment:
Have you tried to build it from source?

Reply:
Was poking around with that for a few days, but wasn't able to get it all to behave properly, so I decided to get a working flatpak up before I continue poking around with a from-source build.

Reply:
makes sense, it should pretty straightforward in most of the cases. Not a blocker though